build.proj

```diff
@@ -10,17 +10,17 @@
   <ItemGroup>
     <Project Include="src\dirs.proj">
       <!-- For the root traversal default filter the OSGroup to the OSEnvironment which is the OS we are running on -->
-      <FilterToOSGroup Condition="'$(BuildAllOSGroups)' != 'true'">$(OSEnvironment)</FilterToOSGroup>
+<!--      <FilterToOSGroup Condition="'$(BuildAllOSGroups)' != 'true'">$(OSEnvironment)</FilterToOSGroup> -->
```

Comment:
Perhaps instead of commenting these out just set the BuildAllOSGroup property to true.